pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -170,6 +170,16 @@ func (pas *PodAutoscalerStatus) IsInactive() bool {
 	return pas.GetCondition(PodAutoscalerConditionActive).IsFalse()
 }
 
+// IsInitiallyActive returns true if the pod autoscaler has reached its initial scale.
+func (pas *PodAutoscalerStatus) IsInitiallyActive() bool {
```

Comment:
could be worth bike-shedding on this a little. Maybe `HasBeenActive` or `HasActivatedOnce`? `IsInitiallyActive` seems to imply it's _currently_ in its first activation, which isn't quite right